Sources/BraintreeCore/Public/BraintreeCore/Braintree.h

```diff
@@ -1,8 +0,0 @@
-#ifndef Braintree_h
-#define Braintree_h
-
-/// This file added to resolve an issue with cross language module support using CocoaPods:
-/// https://github.com/CocoaPods/CocoaPods/issues/9432#issuecomment-570830515
-/// When compiling a Swift module, the `-Swift.h` header references `Braintree.h`, but requires nothing from it.
-
-#endif /* Braintree_h */
```

Comment:
🤔 I think we might need to add this file back for CocoaPods. If we add it back, does this PR (local project) still build? Might be one less thing to hunt down when we follow up with the package managers

Reply:
Yep, based on our findings yesterday we will. Adding back in this PR to keep the size of the follow up PR smaller. Also updating a number of the imports here vs in the follow up PR.